examples/wagtail/core/jinja/home_page.jinja

```diff
@@ -12,7 +12,7 @@
 {% from 'button/button.html' import govieButton %}
 {% from 'icon/icon.html' import govieIcon %}
 {% from 'tag/tag.html' import govieTag %}
-{% from 'radio/radios-group.html' import govieRadiosGroup %}
+{% from 'radio/radios-group.html' import govieRadioGroup %}
```

Comment:
maybe we should remove the examples?

Reply:
I started to work on removing macros and wagtail in another branch